
Fromm bc8d50cb39b925b0bf551a65e2cdf3420c2f77cf Mon Sep 17 00:00:00 2001
12 From: Stivvo <stivvo01@gmail.com>
11 Date: Tue, 15 Sep 2020 13:20:05 +0200
10 Subject: [PATCH] focusmon() moves the cursor accordingly

Moves the cursor to the recently focused monito

Keeps same x and y position relative to the monitor
---
--- ./dwl.c.orig	2021-02-28 13:44:13.837364093 +0200
+++ ./dwl.c	2021-02-28 13:46:25.561511557 +0200
@@ -1167,10 +1167,12 @@
 void
 focusmon(const Arg *arg)
 {
+    Monitor *prevm = selmon;
 	do
 		selmon = dirtomon(arg->i);
 	while (!selmon->wlr_output->enabled);
 	focusclient(focustop(selmon), 1);
+    wlr_cursor_move(cursor, NULL, selmon->m.x - prevm->m.x , 0);
 }
 
 void
